libs/common/src/lib/widgets/widget-generator.component.html

```diff
@@ -240,36 +241,41 @@ <h1>
                           {{ 'widget.generator.advanced.generative-answer-category.citations.help' | translate }}
                         </pa-popover>
                       </div>
-                      <div class="toggle-and-help">
-                        <pa-toggle
-                          labelOnRight
-                          formControlName="generativeModelToggle"
-                          (valueChange)="toggleGenerativeModels()">
-                          {{
-                            'widget.generator.advanced.generative-answer-category.generative-model.label' | translate
-                          }}
-                        </pa-toggle>
-                        <pa-icon
-                          name="info"
-                          [paPopover]="generativeModelToggleHelp" />
-                        <pa-popover #generativeModelToggleHelp>
-                          {{ 'widget.generator.advanced.generative-answer-category.generative-model.help' | translate }}
-                        </pa-popover>
-                      </div>
-                      @if (generativeModelEnabled && generativeModels.length > 0) {
+                      @if (generativeModels.length > 0) {
                         <div class="toggle-and-help">
-                          <pa-select
-                            formControlName="generativeModel"
-                            [label]="
+                          <pa-toggle
+                            labelOnRight
+                            formControlName="generativeModelToggle"
```

Comment:
It seems in a prevous PR we forgot to add `generativeModelToggle` to `notFeatures` list